src/initialize.F90

```diff
@@ -902,4 +916,230 @@ subroutine allocate_banks()
 
   end subroutine allocate_banks
 
+!===============================================================================
+! PREPARE_DISTRIBCELL initializes any distribcell filters present and sets the
+! offsets for distribcells
+!===============================================================================
+
+  subroutine prepare_distribcell()
+
+    integer :: i, j       ! Tally, filter loop counters
+    integer :: extra      ! Number of extra filters to add
+    integer :: n_filt     ! Number of filters originally in tally
+    logical :: count_all  ! Count all cells
+    type(TallyObject),    pointer :: tally            ! Current tally
+    type(Universe),       pointer :: univ             ! Pointer to universe
+    type(Cell),           pointer :: c                ! Pointer to cell
+    integer, allocatable :: univ_list(:)              ! Target offsets
+    integer, allocatable :: kounts(:,:)               ! Target count
```

Comment:
`counts` is not an intrinsic procedure or reserved word in Fortran, so I'd much prefer it over `kounts`.